MVVM/Model/RtspStreamClient.cs

```diff
@@ -12,6 +13,10 @@ namespace RoverControlApp.MVVM.Model
 {
 	public class RtspStreamClient : IDisposable
 	{
+		[Signal]
+		public delegate void FrameReceivedEventHandler(int id);
+		public event FrameReceivedEventHandler? FrameReceived;
```

Comment:
Left it like that, thinking about future multicamera support (I did it like that in multicamera before "the great rebase"). When I'll get to it it would be already as it would have to be (or at least how I would do it xd)

Reply:
Ok then